packages/uniforms-mantine/src/AutoField.tsx

```diff
@@ -0,0 +1,33 @@
+import invariant from 'invariant';
+import { createAutoField } from 'uniforms';
+export { AutoFieldProps } from 'uniforms';
+
+import BoolField from '/client/components/uniforms/base/BoolField';
```

Comment:
Seems like your IDE has mangled all the import paths, will probably need to adjust them all.